src/components/molecules/UiLoader/UiLoader.vue

```diff
@@ -1,40 +1,35 @@
 <template>
-  <component
```

Comment:
R: we should keep <Transition> for v-if and use our custom transition components for other modes `v-show`, `opacity`.

```
<component :is="rootComponent">
  <component  
    :is="loaderComponent"
    v-if="isLoading"
    class="ui-loader"
   ></component>
  <div v-else>
    <slot/>
  </div>
</component>
```
but if I think about that... we have a different template inside `rootComponent`. IMO we need 
```
// UiLoader
<template>
  <component :is="transitionComponent" />
</template>
```
and add `UiLoaderNativeTransition` and `UiLoaderTransition` and keep in it required functions?
WDYT?

Reply:
I think so too. It's a good idea 👍